framework/packages/abstract-std/src/objects/gov_type.rs

```diff
@@ -77,6 +77,7 @@ impl GovernanceDetails<String> {
     pub fn verify(
         self,
         deps: Deps,
+        // TODO: remove!
```

Comment:
I don't think we can remove this.

Reply:
> I don't think we can remove this.

We have deps, we don't need to pass vc

Reply:
What if we don't want to use the deterministic address?

Reply:
> What if we don't want to use the deterministic address?

Where we wouldn't want to use deterministic address?